src/StdCheats.sol

```diff
@@ -569,4 +575,50 @@ abstract contract StdCheats is StdCheatsSafe {
             stdstore.target(token).sig(0x18160ddd).checked_write(totSup);
         }
     }
+
+    function dealERC1155(address token, address to, uint256 id, uint256 give, bool adjust) internal virtual {
+        // get current balance
+        (, bytes memory balData) = token.call(abi.encodeWithSelector(0x00fdd58e, to, id));
+        uint256 prevBal = abi.decode(balData, (uint256));
```

Comment:
Should we check if the call was successful here, like we do in `dealERC721`?

Reply:
Got it, appreciate the explanation! Makes sense, I think it's ok to leave as-is then